.github/workflows/emscripten-experimental.yml

```diff
@@ -0,0 +1,88 @@
+name: 'Test Build (Emscripten Experimental)'
+on:
+  workflow_dispatch:
+  workflow_run:
```

Comment:
`workflow_run` means it will only run after the checks in `main.yml` succeed on `master`. Prevents noise if pushes to `master` have an issue with `clang` compilation, etc.